Fix: Assign default value for confidence_score when NA to ensure DBot compatibility

Prateek-Cyware opened the pull request:
… score compatibility

<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Updated the default value for confidence_score to handle cases where it comes as "NA" from the get-threat-data command. This will now be used to calculate the DBot score accordingly.

## Must have
- [x] Tests
- [x] Documentation

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/demisto/content?pullRequest=40787) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=40787) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/demisto/content?pullRequest=40787) it.</sub>

YaelShamai reviewed (changes requested):
Hey,
The PR looks great, see one comment I wrote regarding RN. 
Let me know if you have any questions.

Prateek-Cyware commented:
> Hey, The PR looks great, see one comment I wrote regarding RN. Let me know if you have any questions.

Hi,
I’ve addressed the comment. Please let me know if you have any further questions or need any additional clarification. thanks!

Prateek-Cyware commented:
Hi @YaelShamai , could you please let me know if any thing else is required from my side?

YaelShamai commented on Packs/CTIX/ReleaseNotes/2_2_20.md:
```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### CTIX v3
+- Set default confidence_score value when received as "NA" in *ctix-get-threat-data*  command to support DBot score calculation.
```
Ok, so we need to change the rn to say something like this:
Fixed an issue in the *ctix-get-threat-data* command that failed when.....

YaelShamai reviewed (changes requested):
Hey, see my comment please.
